src/platform/forms-system/test/js/review/SubmitController.unit.spec.jsx

```diff
@@ -236,6 +249,74 @@ describe('Schemaform review: SubmitController', () => {
 
     expect(submitForm.called).to.be.false;
     expect(setSubmission.calledWith('hasAttemptedSubmit')).to.be.true;
+    expect(setSubmission.calledWith('status', 'validationError')).to.be.true;
```

Comment:
This ensures that the error is triggered because of validation, not because the pre-submit agreement has been accepted.